static/app/views/performance/transactionSummary/transactionVitals/index.spec.tsx

```diff
@@ -40,10 +52,10 @@ function initialize({project, features, transaction, query} = {}) {
   return data;
 }
 
-function WrappedComponent({organization, ...props}) {
+function WrappedComponent({location, organization, ...props}) {
```

Comment:
This needs to be typed

Reply:
good catch! `router` for instance is not a required prop